pages/view-note.html

```diff
@@ -25,8 +25,57 @@ <h1 class="heading3 medium brand-name">Doodles</h1>
+        </button>
+        <section id="view-tool" class="view-tools">
+            <button data-toolClose="view-tool" class="icon-button tool-button-colour m9">
+                <i class="bi bi-caret-right-square"></i>
+            </button>
+            <section class="view-tool-btns">
+                <button class="btn btn-primary outline btn-link td">
+                    <i class="bi bi-cloud-upload inside-btn"></i>Update Note
+                </button>
+                <button class="btn btn-primary outline btn-link td m-t10">
+                    <i class="bi bi-trash3 inside-btn"></i>Delete Note
+                </button>
+            </section>            
+        </section>
+
+        <section class="view-title">
+            <h4 class="text3 regular text-center">Tom N Jerry</h4>
+        </section>
+        <section class="view-content">
+            <p class="text4 regular">The series features comic fights between two lifelong enemies, a cat (Tom) and a mouse (Jerry). The plots of each short usually center on Tom and Jerry's numerous attempts to have the best of each other and the mayhem and destruction that follows. Despite Tom's clever strategies (whether they work or not), determined and energetic mindset, large size, and exceptional overall intelligence, he rarely succeeds in getting the best of Jerry, mainly because of Jerry's cunning abilities, luck, and his lack of tendencies in being a bit too reckless. However, on several occasions, they have displayed genuine friendship and concern for each other's well-being. At other times, the pair set aside their rivalry in order to pursue a common goal, such as when a baby escapes the watch of a negligent babysitter, causing Tom and Jerry to pursue the baby and keep it away from danger, in the shorts Busy Buddies and Tot Watchers respectively. Despite their endless attacks on one another, they have saved each other's lives every time they were truly in danger, except in The Two Mouseketeers, which features an uncharacteristically morbid ending.
+            </p>
+            <br>
+            <p class="text4 regular">
+                The cartoons are known for some of the most violent cartoon gags ever devised in theatrical animation: Tom may use axes, hammers, firearms, firecrackers, explosives, traps and poison to kill Jerry. On the other hand, Jerry's methods of retaliation are far more violent, with frequent success, including slicing Tom in half, decapitating him, shutting his head or fingers in a window or a door, stuffing Tom's tail in a waffle iron or a mangle, kicking him into a refrigerator, getting him electrocuted, pounding him with a mace, club or mallet, letting a tree or electric pole drive him into the ground, sticking matches into his feet and lighting them, tying him to a firework and setting it off, and so on. While Tom and Jerry has often been criticized as excessively violent, there is no blood or gore in any scene.</p>
+                <br>
```

Comment:
any reason to use <br> here instead of handling it in css?

Reply:
Currently haven't defined CSS for content that is seen as note content. Will define it further.